server/config/env/development.js

```diff
@@ -12,6 +12,17 @@ module.exports = {
     // Enable mongoose debug mode
     debug: process.env.MONGODB_DEBUG || false
   },
+  orm: {
```

Comment:
I'm performing changes in chunks so it is easier to follow and process, meaning now I only refactored mongoose to be able to support both that and sequelizejs. When we get to the configuration refactoring we will definitely improve that through-out the codebase.